gatsby-ssr.js

```diff
@@ -13,3 +13,28 @@ export const wrapRootElement = ({ element }) => {
 }
 
 export const wrapPageElement = WrapPagesWithLocaleContext
+
+export const onRenderBody = (
+    { setHeadComponents },
+    { accountId, async = true }
```

Comment:
`accountId` seems to be redundant. do we need it?

Reply:
Thanks for this forgot to keep the account id there. Updated the code